### ISSUE TITLE
The expiration date is deleted when it expires.

### ISSUE THREAD

**Issue by ClappTV** [hrfee/jfa-go#379]
Once the expiration date passes, it is deleted, so you don't know what the date was. It would be good if, even after the date has expired, it was kept so that you would know at all times when it expired.

**Comment by hrfee** [comment]
This information should be in the activity log.

**Comment by iPleyGit** [comment]
> Esta información debería estar en el registro de actividad.

Yes, that’s how I currently use it. I check the expiration date in the log, but for example, when an account expires, if you want to extend it, you have to set the date from scratch since it gets deleted. It would be convenient if the date didn’t get deleted and you could set the time with the default values of 1 month or 3 months without having to enter the full date.

**Comment by iPleyGit** [comment]
When I want to extend a user who hasn’t expired yet, I just need to click on “extend expiration” and add 3 months, and it automatically adds 3 months to their current expiration date.

When a user has expired, the problem is that the expiration date is gone, so if I want to add 3 months to their previous expiration date, I have to enter it manually because there’s no date configured anymore.
